index.html

```diff
@@ -9222,6 +9225,274 @@ <h3 id="example-2">
+          </p>
+          <ol>
+            <li>Let <var>input signal</var> be the the render quantum of audio
+            to be processed.
+            </li>
+            <li>Let <var>threshold</var>, <var>knee</var> have the value of the
+            <a>AudioParam</a> of the same name, <a href=
+            "#db-to-linear">converted to linear unit</a> sampled at the time of
+            processing of this block (as <a>k-rate</a> parameters).
+            </li>
+            <li>Let <var>ratio</var> have the value of the <code>ratio</code>
+            <a>AudioParam</a>, sampled at the time of processing of this block
+            (as a <a>k-rate</a> parameter).
+            </li>
+            <li>Let <var>attack</var> and <var>release</var> have the value of
+            the <a>AudioParam</a> of the same name, sampled at the time of
+            processing (those are <a>k-rate</a> parameters), mutiplied by the
+            sample-rate of the <a>BaseAudioContext</a> this
+            <a>DynamicsCompressorNode</a> is <a href=
+            "#associated">associated</a> with.
+            </li>
+            <li>For each samples <var>input</var> of <var>input signal</var>,
+            execute the following steps:
+              <ol>
+                <li>Let <var>envelope rate</var> be the result of <a href=
+                "#envelope-rate">computing the envelope rate</a>
+                </li>
+                <li>If the absolute value of <var>input</var> is less than
+                0.0001, let <var>attenuation</var> be 1.0. Else, let
+                <var>attenuation</var> be the value of applying the <a
+                href="#compression-curve">compression curve</a> to the absolute
+                value of <var>input</var>, divided by the absolute value of
+                <var>input</var>.
+                </li>
+                <li>Let <var>decibels per frame</var> be the result of dividing
+                <var>attenuationDB</var> by <var>releaseRate</var>.</li>
+                <li>Set <var>relaseRate</var> be the result of
+                <a href="#db-to-linear">converting <var>decibels per frames
+                </var>to linear gain</a>, and substract one.</li>
+                <li>Set <var>change-rate</var> to the difference between <var>
+                  attenuation</var> and <var>detector average</var>.
+                </li>
+                <li>Apply the <a href="#detector-curve">detector curve</a> to
+                <var>change-rate</var></li>
+                <li>Add <var>change-rate</var> to <var>detector average</var>,
+                and clamp <var>detector average</var> to a maximum of 1.0.  </li>
+                <li>If attacking, set <var>attack</var> to <var>detector
+                average</var> minus <var>compressor gain</var>. Set
+                <var>compressor gain</var> to the multiplication of
+                <var>attack</var> by <var>envelope rate</var>. Clamp
```

Comment:
The sentence "Set compressor gain to the multiplication of..." appears to be left over from before. It should be replaced by the new language in line 9382 about multiplying attack by envelope rate and adding attack to compressor gain.